src/software/ai/passing/pass_generator.h

```diff
@@ -36,10 +36,10 @@ namespace Passing
      * == Performance Considerations ==
      * It is important that the pass generator is able to "keep up" with the current
      * time. This is because a pass is defined in part by it's start time. If the
-     * pass generator only completes an interation of pass updates once every 5 seconds,
+     * pass generator only completes an interaction of pass updates once every 5 seconds,
```

Comment:
please revert

Reply:
Sorry, just to clear this up, it's misspelled, but it should be `iteration`, not `interaction`.